src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -2781,8 +2781,14 @@ public MethodHandle findConstructor(Class<?> refc, MethodType type) throws NoSuc
          * to be initialized if it has not been already initialized,
          * as specified in JVMS {@jvms 5.5}.
          *
+         * <p>
+         * This method returns when {@code targetClass} is fully initialized, or
+         * when {@code targetClass} is being initialized if this method is called
+         * by the initializing thread.
```

Comment:
This looks okay but I wonder if it might be a bit clearer to say "when targetClass is being initialized on the current thread".

Reply:
That works for me.  Updated.